### DISCUSSION TITLE
how can I replace the ugly icon on toolbar?

### DISCUSSION THREAD

**Discussion by bonafideyan** [remirror/remirror#1404] [Q&A]
https://docs.remirror.org/showcase/wysiwyg

this is beautiful.

**Comment by tx0c** [comment]
wow, the https://docs.remirror.org whole site looks completely new to me, even I have read https://remirror.io/docs/ for a while, I don't see anywhere of remirror.io has a link to https://docs.remirror.org site, wonder who set that https://docs.remirror.org site up? and where's the code for https://docs.remirror.org/showcase/wysiwyg ?

I'm reaching the same problem here, if want to re-use `const toolbarItems: ToolbarItemUnion[]` from examples, I see it's either `type: ToolbarCommandButton` to use the builtin defined icon, or use `type: ToolbarButton` then set `icon: ...` to something else but this is still limited to all CoreIcon ( from https://github.com/remirror/remirror/blob/main/packages/remirror__icons/src/core-icons.ts ) names?

        { type: ComponentItem.ToolbarCommandButton, commandName: 'toggleCodeBlock', display: 'icon' },
        { type: ComponentItem.ToolbarButton, label: '--', onClick() {
          chain // Begin a chain
            .insertHorizontalRule()
            .focus()
            .run(); // A chain must always be terminated with `.run()`
        } },
        { type: ComponentItem.ToolbarButton, icon: 'linkM', onClick() {
          console.log('linkM onClick:');
        } },

If I'm ok with some basic icons (bold/italic/...), but I want to add some extra buttons with customized svg icons, is that possible ?
